lib/msf/base/sessions/ssh_command_shell_bind.rb

```diff
@@ -200,6 +200,8 @@ def create(cid, ssh_channel, peer_host, peer_port)
         }
       end
 
+      attr_reader :client
```

Comment:
This fixes the missing `via` string when starting a reverse listener from an SSH session.